frontend/src/lib/main.ts

```diff
@@ -14,3 +14,20 @@ export const me: Writable<User> = writable({
 	groups: [],
 	permissions: []
 });
+
+/**
+ * New store for branch name
+ */
+export const branchName: Writable<string> = writable('Set Branch'); // Default branch name
+/**
+ * New store for the tree
+ */
+export const documentTreeStore = writable<INode>({
```

Comment:
These docstrings are a little weird, I don't see why you'd specify that they're new, and they don't really provide any info that the variable name/type annotation doesn't